rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/20_source_mapping.yml

```diff
@@ -25,34 +14,19 @@ stored _source mode is not supported:
           mappings:
             _source:
               mode: stored
-            properties:
```

Comment:
why are these fields removed from the mapping here and also in other places below?

Reply:
Just because it does not matter for this test. They are in default mapping.

Reply:
Ok I see. 👍  I thought that it was somehow related to allowing stored source, but it is not.